### PR TITLE
introduced icon option in tags

### PR DESCRIPTION
**Pull request by rishabhPlivo**

### CODE REVIEW

File: src/components/Tag/index.js [hunk 1]
```diff
@@ -2,15 +2,20 @@ import React from 'react'
...
 import { Text } from '../Typography/index'
+import { Icon } from '../Icon/index'
+import { Block } from '../Block'
 
 export const TAG_TYPES = ['success', 'danger', 'disabled', 'progress', 'default', 'warning']
 /**
  * Tag component that is used to specify status of a task
  */
-export const Tag = ({ className = '', type, label, children, ...props }) => {
+export const Tag = ({ className = '', type, label, children, icon, ...props }) => {
```
Comment (RohanNegiUI) [inline src/components/Tag/index.js:12]: Add icon_svg props as well.

File: src/components/Tag/index.js [hunk 2]
```diff
@@ -2,15 +2,22 @@ import React from 'react'
...
  */
-export const Tag = ({ className = '', type, label, children, ...props }) => {
+export const Tag = ({ className = '', type, label, children, icon, svg, ...props }) => {
+  const SvgIcon = svg
   return (
     <span className={['sg contacto-tag', className, `contacto-tag--${type}`].join(' ')} {...props}>
-      <Text type={type === 'default' ? 'caption' : 'caption-capital'}>{label || children}</Text>
+      <Block display="flex" alignItems="center">
+        {icon && <Icon name={icon} size={14} className="tag-icon" />}
+        {svg && <SvgIcon />}
```
Comment (RohanNegiUI) [inline src/components/Tag/index.js:18]: you need to add svg like below
<Icon svg={svg} size={14} className="tag-icon" />

**Review by RohanNegiUI — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
     "registry": "https://npm.pkg.github.com"
   },
   "repository": "git://github.com/contacto-io/contacto-web-app",
-  "version": "0.1.44",
```
Comment (RohanNegiUI) [inline package.json:7]: change it to 46. I have already published with 45

Comment (rishabhPlivo) [reply package.json:7]: done

**Pull request merged**